packages/evm/package.json

```diff
@@ -48,6 +48,7 @@
   "dependencies": {
     "@ethereumjs/common": "^3.0.0",
     "@ethereumjs/util": "^8.0.0",
+    "@ethersproject/providers": "^5.7.1",
```

Comment:
Is this correct to have this dependency now in the EVM main dependencies? 🤔

Reply:
That's the package within `ethers` that contains the provider interface used by the new State Manager to retrieve data.  So, we either bundle all of `ethers` or that specific package or I suppose I could duplicate the interface as an internal type though that might lead to typing issues for user.  Not sure unless I try it.

Reply:
See other comment on Ethers